src/content/docs/style-guide/writing-strategies/contribute-to-site.mdx

```diff
@@ -0,0 +1,376 @@
+---
+title: Contribute to the docs site 
+contentType: page
+template: basicDoc
+topics:
+  - Basic style guide
+  - Writing guidelines
+redirects:
+  - >-
+    /docs/new-relic-only/style-guide/processes-and-procedures/creating-and-editing-content
+  - >-
+    /docs/new-relic-only/style-guide/processes-procedures/creating-editing-content
+  - >-
+    /docs/new-relic-only/basic-style-guide/basic-style-guide/creating-editing-content
+  - /docs/new-relic-only/basic-style-guide/basic-style-guide/create-edit-content
+  - /docs/style-guide/writing-guidelines/create-edit-content
+  - /docs/style-guide/writing-docs/processes-procedures/create-edit-content
+---
+
+import styleguideEditDoc from 'images/style-guide_screenshot-crop_edit-doc.webp'
+
+import styleguidePencilButton from 'images/style-guide_screenshot-crop_edit-button.webp'
+
+import styleguideCommitChanges from 'images/style-guide_screenshot-crop_edit-commit-changes.webp'
+
+import styleguideCommitMessage from 'images/style-guide_screenshot-crop_edit-commit-message.webp'
+
+import styleguideForkRepo from 'images/style-guide_screenshot-crop_edit-fork-repo.webp'
+
+import styleguideForkModal from 'images/style-guide_screenshot-crop_edit-fork-modal.webp'
+
+import styleguideCloneFork from 'images/style-guide_screenshot-crop_edit-clone-fork.webp'
+
+import styleguidePRpage from 'images/style-guide_screenshot-crop_edit-pr-page.webp'
+
+import styleguideContinueWorkPR from 'images/style-guide_screenshot-crop_edit-files-changed.webp'
+
+import styleguideManageProjects from 'images/style-guide_screenshot-crop_edit-manage-project-status.webp'
+
+import styleguideGHDbranch from 'images/style-guide_screenshot-crop_edit-ghd-new-branch.webp'
+
+import styleguideBaseCompare from 'images/style-guide_screenshot-crop_edit-ghd-base-compare.webp'
+
+
+The docs team welcomes your contributions, whether you're a New Relic employee or a New Relic user. This doc walks you through three different methods for making or requesting changes to the docs site. 
+
+## Prerequisites [#prereq]
+
+Whatever method you choose, you need to sign up for a [GitHub account](https://github.com/join) if you haven't yet. 
+
+GitHub is at the heart of our whole process. When you propose a change to the docs site, GitHub manages the two different versions of a doc: the original version as it exists on docs.newrelic.com and the edited version that exists on your screen. This creates a protective layer that allows the docs team to review and approve changes before they're released to the site. 
```

Comment:
I think you should bold "docs.newrelic.com" or make it a link

Reply:
There's something off to me about "protective layer" in this sentence: "This creates a protective layer that allows the docs team to review and approve changes before they're released to the site."

Maybe make this more active? "With any edit you make, the Docs team reviews and approves it before it's published." Or something like that